Propose changes to works and about myself sections

paulsaenzsucre opened the pull request:
In this pull request a made these changes:

- Modify the index.html file with a My Recent Works and a About Me sections using HTML5.
- Modify the styles.css file with several styles for the My Recent Works and About Me section. Use CSS Grid and Flexbox to lay out the elements and pseudo-classes to implement some basic interactions.
- Write a descriptive README file.
- Include a MIT license.

Also, did the following activities:

- Setup the linters for my project.
- Look for and fix linter errors.
- Create a new branch for coding.
- Do frequents commits with incremental changes.

paulsaenzsucre committed:
Add initial HTML structure and CSS for the My Recent Works part.

HTML changes:

- Add My Recent Works  <article> tag.

- Add inside the <article> tag a <header> tag, with a <h1> and <hr> tags inside, for the My Recent Works headline.

- Add inside the <article> tag six <section> tags, with header>h2>br, ul>li*5 and button tags; for work cards.

CSS changes:

- Change #headline selector to .headline to reuse in other headlines.

- Add pageHeader class for styling the page header.

- Add myRecentWorksContainer id for styling the My Recent Works part.

- Add workCard class for styling the work cards.

paulsaenzsucre committed:
Add styling for the My Recent Works part.

- Add langBadgesContainer class for styling the container of languages badges elements of the work card info.

- Add langBadges class for styling languages badges elements of the work card info.

- Add workCardInfoContainer class for styling the container of work card info elements of the work card.

- Add btnSeeProject class for styling the button of work card info elements of the work card.

- Add workCardInfoTitle class for styling the header of the work card info elements of the work card.

- Add recentWorkSeparator id for styling the separator of the work card info elements of the work card.

paulsaenzsucre committed:
Add images files for the About Me part.

- Add frameworksIcon.svg to images folder.

- Add languagesIcon.svg to images folder.

- Add skillsIcon.svg to images folder.

paulsaenzsucre committed:
Add initial HTML structure and CSS for the About me part

HTML changes:

- Add About Me <article> tag.

- Add inside the <article> tag a <header> tag, with a <h1> tag inside, for the About Me headline.

- Add inside the <article> tag a <section> tag, with a <p> and <button> tags inside for the supporting text of the About Me part.

- Add inside the <article> tag a <section> tag, with h2+ul>li*4 tags; for the languages card of the About Me part.

- Add inside the <article> tag a <section> tag, with h2+ul>li*5 tags; for the frameworks card of the About Me part.

- Add inside the <article> tag a <section> tag, with h2+ul>li*5 tags; for the skills card of the About Me part.

CSS changes:

- Change #supportingLine selector to .supportingLine to reuse in other supporting lines.

- Add aboutMeContainer class for styling the About Me part.

- Add langFrameAndSkillCard class for styling the languages, frameworks and skills cards.

paulsaenzsucre committed:
Update the images files for the About Me part.

- Update frameworksIcon.svg file in images folder.

- Update languagesIcon.svg file in images folder.

- Update skillsIcon.svg file in images folder.

- Add anchorButton.svg to images folder.

paulsaenzsucre committed:
Add HTML and CSS for the About Me part.

HTML changes:

- Add inside the languages, frameworks and skills <section> tags <div> tags for wrap the icons.

Add langFrameAndSkillsIconContainer class for styling the surroundig white circle wrapping languages, frameworks and skilss icons for the cards on the About Me part.

Add langFrameAndSkillsIcon class for styling languages, frameworks and skilss icons for the cards on the About Me part.

Add langFrameAndToolsBadgesContainer class for styling languages, frameworks and skills

Add langFrameAndToolsBadges class for styling the languages, frameworks and skills badges

- Add class langFrameAndSkillsTitle

- Add class btnGetMyResume.

paulsaenzsucre committed:
Fix CSS linter errors

Bondok6 commented on css/styles.css:
```diff
@@ -107,3 +107,202 @@ header {
+  white-space: nowrap;
+}
+
+.btnSeeProject {
+  background-color: #36b37f;
+  border-radius: 4px;
+  color: white;
+  width: 122px;
+  height: 48px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  margin: 0 0 0;
+}
```
- [x] Great job here 💯, But according to the requirements, you should Implement the button interactions (enable, hover, etc.). Based on your Figma design, so Kindly add them 👍

![image](https://user-images.githubusercontent.com/45231276/198760112-9062a83a-08ac-4e75-af99-eb66e30789bf.png)

![image](https://user-images.githubusercontent.com/45231276/198760704-5bd03acc-fa67-4607-93ae-ad2fe70dfc8e.png)

Bondok6 commented on css/styles.css:
```diff
@@ -107,3 +107,202 @@ header {
+  background-position: bottom 23px left -30px;
+}
+
+.btnGetMyResume {
+  background-color: #36b37f;
+  border-radius: 4px;
+  color: white;
+  width: 159px;
+  height: 48px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  margin: 0 0 25px;
+}
```
- [x] The same thing here, kindly check my comment above ⬆
> Please, add the button interactions (hover)
```
.btnGetMyResume:hover {
 ......
}
```

Bondok6 commented on index.html:
```diff
@@ -33,7 +33,156 @@ <h1 id="headline">Hey there. I'm Paul Sáenz
+    </section>
+
+    <article id="myRecentWorksContainer">
+      <header>
+        <h1 class="headline">My Recent Works</h1>
+        <hr id="recentWorkSeparator">
```
- [OPTIONAL]  I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

> 💡 NOTE: you've used the `<h1>` tag once here `<h1 class="headline">Hey there. I'm Paul Sáenz  ... `, and that's great because this is the main title 💯

> **Only Use H1 Elements Once Per Page for optimal SEO**

Bondok6 commented on index.html:
```diff
@@ -33,7 +33,156 @@ <h1 id="headline">Hey there. I'm Paul Sáenz
+    </article>
+
+    <article id="aboutMeContainer">
+      <header>
+        <h1 class="headline">About me</h1>
+      </header>
```
- [OPTIONAL] The same thing here, kindly check my comment above ⬆

Bondok6 reviewed (changes requested):
## Hi @elafildecolor 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media0.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Great job using semantic HTML5 tags ✅
- Great job using CSS Grid and Flexbox ✅


## Required Changes ♻️
- [x] _Check the comments under the review._ ⬇

## Optional suggestions
- [x] _Check the comments under the review._ ⬇

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

paulsaenzsucre committed:
Add some CSS variables for colors and shadows.

Add styling for buttons hover.

Reorder <p> and <button> tags inside header of About Me part for better styling.

Felix45 reviewed (approved):
# Approved :green_circle: 
Hi @elafildecolor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] implemented all the required and optional changes from the previous reviewer :white_check_mark: 

![Congratulations-Memes-FEATURED](https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
